public/app/features/scenes/dashboard/inspector/ScenePanelInspector.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { locationUtil } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+import {
+  SceneComponentProps,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneObject,
+  sceneGraph,
+  VizPanel,
+} from '@grafana/scenes';
+import { Drawer, Tab, TabsBar } from '@grafana/ui';
+import { supportsDataQuery } from 'app/features/dashboard/components/PanelEditor/utils';
+
+import { InspectDataTab } from './InspectDataTab';
+import { InspectStatsTab } from './InspectStatsTab';
+import { InspectTabState } from './types';
+
+interface ScenePanelInspectorState extends SceneObjectState {
+  tabs?: Array<SceneObject<InspectTabState>>;
+}
+
+export class ScenePanelInspector extends SceneObjectBase<ScenePanelInspectorState> {
+  static Component = ScenePanelInspectorRenderer;
+  // Not stored in state as this is just a reference and it never changes
+  private _panel: VizPanel;
+
+  constructor(panel: VizPanel) {
+    super({});
+
+    this._panel = panel;
+    this.buildTabs();
+  }
+
+  buildTabs() {
+    const plugin = this._panel.getPlugin();
+    const tabs: Array<SceneObject<InspectTabState>> = [];
+
+    if (!plugin) {
+      // TODO temp fix
+      setTimeout(() => this.buildTabs(), 100);
+      return;
+    }
```

Comment:
There is a tricky case here if you inspect a panel that is below the fold (so loaded lazily), if you do a full page reload of the page the inspect drawer will not show. I just tested this for our current dashboard page and the same bug happens there you cannot load an inspect drawer from url for a panel that is below the fold (scroll to), as the plugin has not been loaded yet.